client/src/components/ConfigTemplates/InstanceDropdown.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import "./icons";
+
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { BLink } from "bootstrap-vue";
+import { useRouter } from "vue-router/composables";
+
+interface Props {
+    prefix: string;
+    name: string;
+    routeEdit: string;
+    routeUpgrade: string;
+    isUpgradable: boolean;
+}
+
+const title = "";
+
+const router = useRouter();
+
+defineProps<Props>();
+
+const emit = defineEmits<{
+    (e: "remove"): void;
+}>();
+</script>
+
+<template>
+    <div>
+        <BLink
+            v-b-tooltip.hover
+            :class="`${prefix}-instance-dropdown font-weight-bold`"
+            data-toggle="dropdown"
+            :title="title"
+            aria-haspopup="true"
+            aria-expanded="false">
+            <FontAwesomeIcon icon="caret-down" />
+            <span class="instance-dropdown-name">{{ name }}</span>
+        </BLink>
+        <div class="dropdown-menu" :aria-labelledby="`${prefix}-instance-dropdown`">
+            <a
+                v-if="isUpgradable"
+                class="dropdown-item"
+                @keypress="router.push(routeUpgrade)"
+                @click.prevent="router.push(routeUpgrade)">
+                <span class="fa fa-edit fa-fw mr-1" />
```

Comment:
Can the `FontAwesomeIcon` component be used here?
Class based icons increase bundle size significantly, so it would be nice if we could get rid of them eventually.